assets/js/membraneWebRTC.ts

```diff
@@ -711,15 +712,23 @@ export class MembraneWebRTC {
   }
 
   /**
+   * Currently, this function only works when DisplayManager in RTC Engine is
+   * enabled and simulcast is disabled.
+   *
    * Prioritizes a track in connection to be always sent to browser.
+   *
```

Comment:
Again, I would swap order of sentences and prioritize saying what this function does

Reply:
If you want to make sure that it's seen, maybe a simple
```
*WARNING!* Only works when DisplayManager is enabled...
```
would do the trick?